package.json

```diff
@@ -23,7 +23,7 @@
   },
   "homepage": "https://carbon-addons-iot-react.com",
   "scripts": {
-    "build": "cross-env BABEL_ENV='production' NODE_ENV='production'  yarn build:pre && rollup -c && yarn build:post",
+    "build": "yarn svg-convert && cross-env BABEL_ENV='production' NODE_ENV='production' yarn build:pre && rollup -c && yarn build:post",
```

Comment:
Could we also run this on pre-commit so we don't have to remember to run it? Below in this file would be 

```diff
 "husky": {
     "hooks": {
-      "pre-commit": "yarn lint-staged && yarn test:base",
+      "pre-commit": "yarn svg-convert && yarn lint-staged && yarn test:base",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
```

Reply:
Added to pre-commit